src/Feature/Changelog/Feature_Changelog.re

```diff
@@ -189,17 +199,79 @@ module View = {
     ];
 
     let error = (font: UiFont.t) => [fontFamily(font.fontFile)];
+
+    let breakingChangeIcon = [width(breakingChangeIconWidth)];
+
+    let caret = [width(caretWidth), marginTop(2)];
+
+    module MoreInfo = {
+      let main = [
+        marginTop(4),
+        marginLeft(
+          typWidth + scopeWidth + breakingChangeIconWidth + caretWidth,
+        ),
```

Comment:
It might be better to just put the whole expanded block in the summary column and let the layout engine do this instead of adding up all the column widths manually.